pkg/templatelookup/modulereleasemeta_test.go

```diff
@@ -0,0 +1,54 @@
+package templatelookup_test
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/require"
+
+	"github.com/kyma-project/lifecycle-manager/api/v1beta2"
+	"github.com/kyma-project/lifecycle-manager/pkg/templatelookup"
+)
+
+func Test_GetChannelVersionForModule_WhenEmptyChannels(t *testing.T) {
+	moduleReleaseMeta := &v1beta2.ModuleReleaseMeta{
+		Spec: v1beta2.ModuleReleaseMetaSpec{
+			Channels: nil,
+		},
+	}
+	_, err := templatelookup.GetChannelVersionForModule(moduleReleaseMeta, "test")
+
+	require.ErrorContains(t, err, "no channels found for module")
```

Comment:
is it important to defence this content of error not get changed? why not assert against `errChannelNotFound`